.github/dependabot.yml

```diff
@@ -1,17 +1,22 @@
 version: 2
 updates:
-- package-ecosystem: maven
-  directory: "/"
-  schedule:
-    interval: daily
-    timezone: Europe/Berlin
-  ignore:
-  - dependency-name: org.antipathy:mvn-scalafmt_2.12
-    versions:
-    - ">= 0.9_1.a, < 0.9_1.1000004"
-  - dependency-name: org.antipathy:mvn-scalafmt_2.12
-    versions:
-    - ">= 1.0.a, < 1.1"
-  - dependency-name: com.mycila:license-maven-plugin
-    versions:
-    - "4.0"
+  - package-ecosystem: maven
+    directory: "/"
```

Comment:
❓ Any particular reason why the indentation changed? Was it not correctly formatted yaml? Is it currently formatted correctly?

Reply:
Exactly, it wasn't properly formatted before.